config/deploy.rb

```diff
@@ -79,6 +79,28 @@
       end
     end
   end
+
+  # You can/ should apply this command to a single host
+  # cap --hosts=pdc-describe-staging1.princeton.edu staging application:remove_from_nginx
```

Comment:
Would it be worthwhile/possible to add a check that the host parameter has been passed (like we do with the nginx playbook) to make sure you don't accidentally take both/all machines down?

Reply:
Is it not ok to take everything off line? I guess I would agree I would not normally want to... I'm not entirely sure how to do it, but I can look into it...

Reply:
Just a thought, definitely doesn't need to be a blocker!